package.json

```diff
@@ -19,10 +19,8 @@
     "prepare": "husky"
   },
   "dependencies": {
-    "@docusaurus/core": "3.1.1",
```

Comment:
`@docusaurus/core` is the only one I would keep because it's usually recommended to list the dependencies that we use directly, even if they're also transitive dependencies. This one is used by `babel.config.cjs`